core/src/api/api.rs

```diff
@@ -255,6 +262,24 @@ impl GraphAPI for GraphState {
 			.borrow()
 			.find_users_without_keys(all_connections.into_iter().collect()))
 	}
+
+	/// Get a list of all private friendship connections that are only valid from users side
+	fn get_one_sided_private_friendship_connections(
```

Comment:
Majority of these api tests should be moved into integration tests and didn't implement a new test in api for this since it would be an integration test setup and we will do it in #42 for all apis

Reply:
I was thinking the same thing; most of the top-level `api` tests are integration-level. 👍🏻